tests/unit/spring_boot_patch.py

```diff
@@ -49,22 +123,27 @@ def list_files(self, path: str):
             file_list.append(file_info)
         return file_list
 
-    def isdir(self, path: str):
+    def isdir(self, path: str) -> bool:
```

Comment:
We have a new setting for `mypy` in the contributing guide that will help spot these (although not in test files): https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#type-hints

Reply:
Added, thanks!